admin/views/licenses.php

```diff
@@ -167,54 +151,60 @@
 					</li>
 				</ul>
 			<?php endif; ?>
-			<?php if ( $extension_list->is_installed( $extension->get_title() ) ) : ?>
+			<?php if ( $extension_list->is_installed( $premium_extension->get_title() ) ) : ?>
 				<div class="yoast-button yoast-button--noarrow yoast-button--extension yoast-button--extension-installed"><?php esc_html_e( 'Installed', 'wordpress-seo' ); ?></div>
 
-				<?php if ( $extensions->is_activated( 'wordpress-seo-premium' ) ) : ?>
+				<?php if ( $has_valid_premium_subscription ) : ?>
 					<div class="yoast-button yoast-button--noarrow yoast-button--extension yoast-button--extension-activated"><?php esc_html_e( 'Activated', 'wordpress-seo' ); ?></div>
-					<a target="_blank" href="<?php WPSEO_Shortlinker::show( 'https://yoa.st/13k' ); ?>" class="yoast-link--license"><?php
+					<a target="_blank" href="<?php WPSEO_Shortlinker::show( 'https://yoa.st/13k' ); ?>"
+					   class="yoast-link--license"><?php
 						/* translators: %s expands to the extension title */
-						printf( esc_html( 'Manage your %s subscription on MyYoast', 'wordpress-seo' ), $extension->get_title() );
+						printf( esc_html( 'Manage your %s subscription on MyYoast', 'wordpress-seo' ), $premium_extension->get_title() );
 						echo $new_tab_message;
-					?></a>
+						?></a>
```

Comment:
I believe this indentation is off

Reply:
For all these things. PHPstorm decided to alter some things for me while I was editing this file. Then I did an autofix, which resulted in this. So might well be right here :)